ykman-gui/qml/SnackBarError.qml

```diff
@@ -27,6 +27,8 @@ SnackBar {
             return qsTr('No YubiKey present')
         case 'open_device_failed':
             return qsTr("Failed to open the application on the YubiKey")
+        case 'windows_admin_required':
+            return qsTr("Failed to open, try to run the application as administrator")
```

Comment:
How about this?
```suggestion
            return qsTr("Failed to open YubiKey. Try running the application as administrator.")
```

Reply:
That was my initial error message. However, after discussing with @braathen we decided the current one was better. What do you think? We don't have error messages with "." in them, so the one you're proposing looks a bit weird in contrast to the other. But I'm not sure, maybe that isn't particularly important.

Reply:
Ok!